apps/wallet/src/features/Widget/BuiltInNetwork/AptosTokensTab/PendingTokens.tsx

```diff
@@ -1,12 +1,12 @@
 import type { FC } from 'react';
 import { Image, StyleSheet } from 'react-native';
+import type { AptosPendingToken } from '@walless/core';
 import { shortenAddress } from '@walless/core';
-import type { AptosPendingToken } from '@walless/engine';
-import { aptosState } from '@walless/engine';
 import { Button, Text, View } from '@walless/gui';
 import type { aptosHandler } from '@walless/kernel';
 import { RequestType } from '@walless/messaging';
 import { showRequirePasscodeModal } from 'modals/RequirePasscode';
+import { aptosState } from 'state/assets';
```

Comment:
use one single convention, sometimes import from `state` this place use `state/assets`